feature_engine/transformation/arcsin.py

```diff
@@ -0,0 +1,170 @@
+# Authors: Soledad Galli <solegalli@protonmail.com>
```

Comment:
either your name, or no name at all. Up  to you :)

Reply:
Modified